src/wallet/scriptpubkeyman.cpp

```diff
@@ -491,6 +519,11 @@ const CKeyMetadata* LegacyScriptPubKeyMan::GetMetadata(const CTxDestination& des
     return nullptr;
 }
 
+uint256 LegacyScriptPubKeyMan::GetID() const
+{
+    return uint256S("0000000000000000000000000000000000000000000000000000000000000001");
```

Comment:
Since this value of one is used in a couple of other places,  I've added a commit refactor those uses and adding a constant `UINT256_ONE` to uint256.h. So we use that constant and so do the other places that use that value of one.